static/src/javascripts/bootstraps/video-embed.js

```diff
@@ -67,6 +71,10 @@ define([
         bean.on(clickbox, 'dblclick', events.dblclick.bind(player));
     }
 
+    function addTitleBar() {
+        $('.vjs-control-bar').after('<div class="vjs-title-bar"><a href="http://www.theguardian.com/' + config.page.pageId + '" target="_parent" class="vjs-title" data-link-name="embed-to-guardian">' + config.page.webTitle + '</a><a href="http://www.theguardian.com/' + config.page.pageId + '" target="_parent" class="vjs-control-content" data-link-name="embed-to-guardian">' + svgs('marque36icon') + '</a></div>');
```

Comment:
As discussed please use the Omniture modules [`logTag()`](https://github.com/guardian/frontend/blob/master/static/src/javascripts/projects/common/modules/analytics/omniture.js#L76) method instead of a campaign code.